include/anvill/MCToIRLifter.h

```diff
@@ -136,7 +142,14 @@ class MCToIRLifter {
                                remill::Instruction *delayed_inst,
                                llvm::BasicBlock *block, bool on_taken_path);
 
-  void VisitInstruction(remill::Instruction &inst, llvm::BasicBlock *block);
+  void VisitInstruction(
+      remill::Instruction &inst, llvm::BasicBlock *block,
+      const std::unordered_map<uint64_t, TypedRegisterDecl> &reg_map);
+
+  llvm::FunctionCallee
```

Comment:
We probably want to return an `llvm::Function *` as I think these function callee APIs things have been changing over time.